tests/forms_tests/tests/test_regressions.py

```diff
@@ -220,3 +220,23 @@ class Meta:
         obj = form.save()
         obj.name = 'Camembert'
         obj.full_clean()
+
+
+class BoundFieldRegresssionsTestCase(TestCase):
+    def test_slice_26267(self):
```

Comment:
I guess I'd put this in tests/forms_tests/tests/test_forms.py with the other `BoundField` tests (ideally, we'd group those in a separate file at some point). The historical grouping of "normal" tests and "regression" tests isn't really useful and we've been slowly removing it.